docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -0,0 +1,72 @@
+# Signalen and the Standard for Public Code
```

Comment:
General remark, there are some spelling issues in the document, like "bugracking"

Reply:
D'oh! I missed that. Thank you. Should be addressed in the follow-on commit.